tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -190,6 +281,19 @@ void test_vector_algorithms() {
     test_find<long long>(gen);
     test_find<unsigned long long>(gen);
 
+    test_min_max_element<char>(gen);
+    test_min_max_element<signed char>(gen);
+    test_min_max_element<unsigned char>(gen);
+    test_min_max_element<short>(gen);
+    test_min_max_element<unsigned short>(gen);
+    test_min_max_element<int>(gen);
+    test_min_max_element<unsigned int>(gen);
+    test_min_max_element<long long>(gen);
+    test_min_max_element<unsigned long long>(gen);
+    test_min_max_element<float>(gen);
+    test_min_max_element<double>(gen);
+    test_min_max_element<long double>(gen);

```

Comment:
this is good, but we need to test the actual special cases. So we need completely equal vectors, and vectors that are have equal chunks, etc. Also vectors that use up whole portions, and vectors with duplicate values in congruent positions (mod chunk size).